test/unit/utils/validations/fields-validations.test.ts

```diff
@@ -116,10 +116,20 @@ describe('fields-validations', () => {
     });
 
     it('can have fields that are not part of date', () => {
-      const validDate = { day: '1', month: '1', year: '2019', saveAndContiune: 'saveAndContiune' };
+      const validDate = { day: '1', month: '1', year: '2000', saveAndContiune: 'saveAndContiune' };
       const validations = dateValidation(validDate, errors);
       expect(validations).to.deep.equal(null);
     });
+
+    it('date must of a person over 18', () => {
+      const notValidDate = { day: '6', month: '2', year: '2002' };
```

Comment:
This test would fail in a few days? Build a date programmatically making sure that it would always be underage